commands/core/cli.drush.inc

```diff
@@ -1,25 +1,9 @@
 <?php
 
-use Drush\Log\LogLevel;
-use Drupal\Component\Assertion\Handle;
-use Drush\Psysh\DrushHelpCommand;
-use Drush\Psysh\DrushCommand;
-use Drush\Psysh\Shell;
-
 /**
  * Implements hook_drush_command().
  */
 function cli_drush_command() {
-  $items['core-cli'] = array(
-    'description' => 'Open an interactive shell on a Drupal site.',
-    'remote-tty' => TRUE,
-    'aliases' => array('php'),
-    'bootstrap' => DRUSH_BOOTSTRAP_MAX,
-    'topics' => array('docs-repl'),
```

Comment:
We are losing this too. Any thoughts on how topics should be handled going forward? I like the current approach but am happy to change. Anyway, not a blocker for this PR.